azure_li_services/schema.py

```diff
@@ -28,7 +28,7 @@
         }
     },
     'networking': {
-        'required': True,
+        'required': False,
```

Comment:
Hmm, I didn't mean to imply that networking should NOT be required. I can't imagine a scenario where an SAP box wouldn't have access to something. Why this change?